docker-compose.yml

```diff
@@ -1,81 +1,59 @@
-version: '2.1'
-
-networks:
-  get5:
-    external: true
+version: "3.7"
 
 services:
-  caddy:
-    image: caddy:2-alpine
-    container_name: caddy-reverse-proxy
-    restart: unless-stopped
+  redis:
+    image: redis:6
+    container_name: redis
+    restart: always
     ports:
-      - "80:80"
-      - "443:443"
-    volumes: 
-      - ./Caddyfile:/etc/caddy/Caddyfile
-    networks:
```

Comment:
Any reason for removing the networks as well? I believe this was needed in order to operate between G5V/Caddy/Redis?

Reply:
I believe docker compose creates a network automatically and manual networks are not needed unless creating multiple networks inside the compose file.